localisation/00_zuwc_united_localisation.csv

```diff
@@ -1209,12 +1209,11 @@ EVTDESC_WCTHW_2010;The people who have become infected with the plague cannot be
+EVTDESC_WCTHW_2011_arthas;Word of your deeds has spread far and wide to foreign courts. The Scourge has driven people such as you to carry out atrocious, but necessary, acts... You and your army have killed all of the infected residents of [From.GetName]. Among the infected people were men, women, and even children, but you showed them no mercy. Following the butchering, you had your men set the city on fire, leaving it in ruins to prevent the dead from ever rising again.;;Die Nachricht von Ihren Taten hat die Hof erreicht. Die Gei�el bringt die Menschen dazu, schreckliche aber notwendig Dinge zu tun.\nDu und deine Armee haben die infizierten Einwohner [From.GetName] abgeschlachtet. Unter ihnen waren M�nner, Frauen und Kinder, aber du zeigte kein Erbarmen. Nach dieser schrecklichen Tat setzte du die Stadt in Brand und legte sie in Schutt und Asche um zu verhindern, dass die Toten wieder auferstehen.;;;;;;;;;;;x
 EVTOPTA_WCTHW_2011_case_01;I can�t believe it!;Je ne peux pas le croire�!;Ich kann es nicht fassen!;;;;;;;;;;;x
 EVTOPTA_WCTHW_2011_case_02;It was the only way out.;;Es war der einzige Ausweg.;;;;;;;;;;;x
-EVTDESC_WCTHW_2015;The Dreadlord [target_dreadlord.GetTitledFirstName], who was involved with the plague of the Scourge in your kingdom, fled to Northrend. There's no doubt that the Scourge came from there. If you strike at the heart of the Scourge, then you might save your people. Upon returning from this campaign, surely you will be greeted as a hero!;;Der Schreckenslord [target_dreadlord.GetTitledFirstName], der an den Gr�ueltaten der Gei�el in Eurem Heimatland beteiligt war, floh nach Northrend. Es gibt keinen Zweifel mehr, dass die Gei�el von dort kam. Wenn Ihr im Herzen der Gei�el zuschlagt, k�nnt Ihr Euer Volk vielleicht noch retten. Wenn Ihr von dieser Kampagne zur�ckkehrt, werdet Ihr gewiss als Held gefeiert werden.;;;;;;;;;;;x
+EVTDESC_WCTHW_2015;The Dreadlord [target_dreadlord.GetTitledFirstName], who was involved with the Scourge's plague in your kingdom, has fled to Northrend. There's no doubt that the source of the Scourge and its terrible sickness originates from there. If you strike at the heart of the Scourge itself, then you may be able to save your people. Upon returning from this campaign, your countrymen and women will surely greet you as a hero!;;Der Schreckenslord [target_dreadlord.GetTitledFirstName], der an den Gr?ueltaten der Gei?el in Eurem Heimatland beteiligt war, floh nach Northrend. Es gibt keinen Zweifel mehr, dass die Gei?el von dort kam. Wenn Ihr im Herzen der Gei?el zuschlagt, k?nnt Ihr Euer Volk vielleicht noch retten. Wenn Ihr von dieser Kampagne zur?ckkehrt, werdet Ihr gewiss als Held gefeiert werden.;;;;;;;;;;;x
```

Comment:
Please change german to "Der Schreckenslord [target_dreadlord.GetTitledFirstName], der an den Gräueltaten der Geißel in Eurem Heimatland beteiligt war, floh nach Northrend. Es gibt keinen Zweifel mehr, dass die Geißel von dort kam. Wenn Ihr im Herzen der Geißel zuschlagt, könnt Ihr Euer Volk vielleicht noch retten. Wenn Ihr von dieser Kampagne zurückkehrt, werdet Ihr gewiss als Held gefeiert werden."

Reply:
Got it.